group_vars/geniza_qa/vars.yml

```diff
@@ -0,0 +1,3 @@
+solr_url: 'https://test-index.cdh.princeton.edu/'
+solr_core: 'geniza_qa'
+gitref: 'experiment/search'
```

Comment:
Let's add a comment here about this. I'm not sure exactly what to put and we're unlikely to leave it for too long, but just as a reminder for ourselves. Maybe something like, "default to flask search app while prototyping" ?